commands/secret_create.go

```diff
@@ -40,6 +41,13 @@ func init() {
 }
 
 func preRunSecretCreate(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return fmt.Errorf("give a name of a secret")
```

Comment:
This might be better as "secret name required", the current language sounds like the secret should already exist

Reply:
Let's please fix this